stock_no_negative/models/stock_location.py

```diff
@@ -0,0 +1,15 @@
+# © 2018 Eficent (https://www.eficent.com)
+# @author Jordi Ballester <jordi.ballester@eficent.com.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import models, fields
+
+
+class StockLocation(models.Model):
+    _inherit = 'stock.location'
+
+    allow_negative_stock = fields.Boolean(
+        string='Allow Negative Stock',
+        help="Allow negative stock levels for the stockable products "
+        "attached to this location. The options doesn't apply to products "
```

Comment:
I cannot understand the second sentence of the help message.

Reply:
@jbeficent what about this?

Reply:
Any news about this?

Reply:
@pedrobaeza @alexis-via I have fixed this last sentence that was wrong.